cluster/config-defaults.yaml

```diff
@@ -27,6 +27,12 @@ cluster_autoscaler_max_graceful_termination_sec: "1209600" # 2 weeks
 # Prevent CA lock-ups caused by large amounts of pending pods. Can be disabled completely by setting to 0.
 cluster_autoscaler_max_usnchedulable_pods_considered: "1000"
 
+# karpenter settings
+karpenter_controller_cpu: "1000m"
+karpenter_controller_memory: "1Gi"
```

Comment:
Maybe we can run with a lower default and use a VPA to scale in bigger clusters? Or we run it on the master nodes 🤔

Reply:
I will double-check the recommendations on karpenter side. VPA sounds good. and yes it is also schedulable on master nodes

Reply:
the recommendations have changed in the recent versions
```
  # We usually recommend not to specify default resources and to leave this as a conscious
  # choice for the user. This also increases chances charts run on environments with little
  # resources, such as Minikube.
  ```
  I will then lower the resource reservation and add VPA

Reply:
added